Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "attestation-agent"
+version = "1.0.0"
+authors = ["The Attestation Agent Authors"]
+publish = false
+edition = "2018"
+
+[dependencies]
+grpcio = { version = "0.6.0", optional = true }
+protobuf = "~2"
+futures = "0.3.5"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+aes-gcm = "0.9.2"
+base64 = "0.13.0"
+clap = "2"
+anyhow = "1.0"
+lazy_static = "1.4.0"
+string-error = "0.1.0"
+
+[build-dependencies]
+protoc-grpcio = { version = "3.0.0", optional = true }
+
+[features]
+default = ["grpc"]
```

Comment:
Should gRPC even be a feature? When will we have an attestation agent that does not run a gRPC service?

Moreover, this initial code would not do much if the grpc feature would be disabled. So I propose that we remove that feature until:
1. There is an actual use case for a grpc-less attestation agent.
2. Someone implements a CLI that would actually allow for passing some input payload through a CLI.